lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -86,9 +90,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
           'crop_rotate' => $values['crop_rotate'],
         ]),
   ];
-
-  $response = drupal_http_request($client['base_url'] . '/reportbacks', $options);
-
+  $response = post($client['base_url'] . '/reportbacks', $options);
```

Comment:
so `post()` doesn't exist standalone like this. You would need to call it off the instantiated `$client` so something more like `$client->post(...)`